README.md

```diff
@@ -18,22 +18,28 @@ SIWE provides a `SiweMessage` class which implements EIP-4361.
 
 Parsing is done by initializing a `SiweMessage` object with an EIP-4361 formatted string:
 
-``` python
+```python
 from siwe import SiweMessage
-message: SiweMessage = SiweMessage(message=eip_4361_string)
+message: SiweMessage = SiweMessage.from_message(message=eip_4361_string)
 ```
 
 Alternatively, initialization of a `SiweMessage` object can be done with a dictionary containing expected attributes:
 
-``` python
-message: SiweMessage = SiweMessage(message={"domain": "login.xyz", "address": "0x1234...", ...})
+```python
+message: SiweMessage = SiweMessage.from_message(message={"domain": "login.xyz", "address": "0x1234...", ...})
```

Comment:
this dict example can be removed now. Also a nitpick for these examples, but the `: SiweMessage` type hint is not needed for all these